blueocean-core-js/src/js/jwt.js

```diff
@@ -41,7 +41,8 @@ export default {
      */
     verifyToken(token, certObject) {
         return new Promise((resolve, reject) =>
-            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg] }, (err, payload) => {
+            // Allow the clocks to be 60s out of sync.
+            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg], clockTolerance: 60 }, (err, payload) => {
```

Comment:
@vivek Well, I wasn't actually going to commit that change as part of this PR since it has nothing to do with IE. I just did it for you guys so ye can see and use.

Also, I would expand on this a bit inside our jwt code i.e. catch the `NotBeforeError` (or perform a time tolerance check ourselves - I'd just catch the `NotBeforeError`) and display something like "Sorry, your machines clock is out of sync with the server clock. You need to ... blah blah blah ....".